.travis.yml

```diff
@@ -44,6 +44,10 @@ script:
   - cd tests/
   - sh -c "if [ '$JSTESTS' != '1' -a '$CODECHECK' != '1' -a '$CODECHECK' != '2' ]; then phpunit --configuration phpunit.xml; fi"
 
+  # Integration tests
+  - cd tests/integration
```

Comment:
we are already in `tests/` and afterwards we need to go back to it for the coverage command?